funnel/templates/layout.html.jinja2

```diff
@@ -239,6 +265,77 @@
       });
     }
   </script>
+
+  <script type="text/javascript">
+    const Store = {
+      // Local storage can only save strings, so value is converted into strings and stored.
+      add(key, value) {
+        return localStorage.setItem(key, JSON.stringify(value));
+      },
+      // Reads from LocalStorage.
+      read(key) {
+        return JSON.parse(localStorage.getItem(key));
+      },
+    };
```

Comment:
LocalStorage must be purged on logout. Needs a method here and/or a comment to guide future maintainers about related places to update.